.travis.yml

```diff
@@ -17,10 +17,12 @@ install:
   - pip install $DJANGO
   - python setup.py -q develop
   - pip install -r requirements-test.txt
+  # TODO remove after openwisp-utils release
+  - pip install --upgrade https://github.com/openwisp/openwisp-utils/tarball/master#egg=openwisp_utils[qa]
 
 script:
-  - openwisp-utils-qa-checks --migration-path "./openwisp_network_topology" --migration-module topology
   - coverage run --source=openwisp_network_topology runtests.py
+  - bash run-qa-checks
```

Comment:
I would have preferred `./run-qa-checks` for consistency, if you can, please update this.

Reply:
Marked in todo, I'll create a small PR with this change! :smile: